packages/react-scripts/fixtures/kitchensink/src/features/syntax/ClassProperties.test.js

```diff
@@ -5,6 +5,8 @@ import ClassProperties from './ClassProperties';
 describe('class properties', () => {
   it('renders without crashing', () => {
     const div = document.createElement('div');
-    ReactDOM.render(<ClassProperties />, div);
+    return new Promise(resolve => {
+      ReactDOM.render(<ClassProperties notifyRendered={resolve} />, div);
```

Comment:
It would be best if you could pass `resolve` as third argument to `ReactDOM.render`.
It is going to become async in the future.